README.md

```diff
@@ -17,7 +17,35 @@ Use this orb to load secrets from SecretHub into your CircleCI jobs.
 
 ## Usage
 
-To execute a command with secrets loaded from SecretHub, install the CLI and set it as the `shell` of the `run` command:
+To execute a command that needs secrets, replace your CircleCI `run` command with `secrethub/exec`.
+You can make secrets available to your command as environment variables by referencing their SecretHub path, prefixed by `secrethub://`:
```

Comment:
Maybe introduce first what this orb does, then explain how to do it? How about switching these two lines?

Reply:
That's how it is now, right? `execute a command that needs secrets` on line 1 is the 'what', `by referencing their SecretHub path` on line 2 is the how.

Reply:
Yes, you're right.

I was looking for a bit longer explanation than "execute a command that needs secrets", but maybe that's enough.